.genx/templates/web-components/route.hbs

```diff
@@ -10,8 +11,18 @@ import { {{pascalCase route.name}}Template as template } from './{{kebabCase rou
 })
 export class {{pascalCase route.name}} extends GenesisElement {
   @User user: User;
+  {{#if route.dynamicLayout}}
+    layout: FoundationLayout;
+    async connectedCallback() {
+      super.connectedCallback();
+      this.loadGenesisCreateLayout();
+    }
 
-  constructor() {
-    super();
-  }
+    loadGenesisCreateLayout() {
+      const dynamicLayout = {{{route.dynamicLayout}}} as unknown as SerialisedLayout;
+      if (!this.layout.tryLoadLayoutFromLocalStorage()) {
+        this.layout.loadLayout(dynamicLayout);
+      }
```

Comment:
This is the best flow I can think of but I think would override when you purposely change the cached layout to pick up new layout. Or did you test it ok?